autopilot: # node name.
  ros__parameters:
    alt_hz: 10.0
    alt_toz: 5.0
    tau: 50.0
    c_kp: 3.0
    c_kd: 0.0
    c_ki: 0.0
    r_kp: 0.75
    r_kd: 0.1
    r_ki: 0.0
    p_kp: -0.5
    p_kd: -0.095
    p_ki: 0.0
    p_ff: 0.0
    a_t_kp: 0.05
    a_t_kd: 0.0
    a_t_ki: 0.005
    a_kp: 0.1
    a_kd: 0.0
    a_ki: 0.01
    l_kp: .5
    l_kd: 0.0
    l_ki: 0.005
    e_kp: .5
    e_kd: 0.0
    e_ki: .01
    trim_e: 0.075
    trim_a: 0.0
    trim_r: 0.0
    trim_t: 0.5
    max_e: 0.61
    max_a: 0.60
    max_r: 0.523
    max_t: 1.0
    pwm_rad_e: 1.0
    pwm_rad_a: 1.0
    pwm_rad_r: 1.0
    max_takeoff_throttle: 1.0
    mass: 4.5
    gravity: 9.8
    max_roll: 35.0
    frequency: 100

path_manager:
  ros__parameters:
    R_min: 100.0
    orbit_last: False
    default_altitiude: 50.0
    default_airspeed: 25.0

path_follower:
  ros__parameters:
    chi_infty: 0.5
    k_orbit: 4.0
    k_path: 0.05

path_planner:
  ros__parameters:
    num_waypoints_to_publish_at_start: 5